packages/@n8n/nodes-langchain/nodes/memory/MemoryRedisChat/MemoryRedisChat.node.ts

```diff
@@ -155,6 +155,7 @@ export class MemoryRedisChat implements INodeType {
 
 		const memory = new memClass({
 			memoryKey: 'chat_history',
+			//@ts-ignore
```

Comment:
Only ts ignore I had. Very weird because I did not touch anything related to this. Wondering if I have to update the lock file 🤔

Reply:
It seems like this was the case! Updated the pnpm-lock.yaml to have we have in master and it seems to have fixed the type issue 🙏